self-reflection-1-skygennette.qmd

```diff
@@ -0,0 +1,63 @@
+
+I have used R since I was an undergrad, and I now use it all the time. I am comfortable coding in R and like the community and resources it comes with. My biggest obstacle is that the packages I have worked with are somewhat limited, so I don't have a great idea of everything that is possible in R. 
+
+6.  Describe your experience working with spatial data and GIS (either in R, or ArcGIS, or QGIS etc.). How do you tend to use spatial data (e.g., making maps, desiging field sampling, buidling stastical models)?
+
+I have worked a lot with many forms of spatial data in R, ArcGIS, and QGIS. I have mostly worked with wildlife movement data, landcover datasets, and remotely sensed spectral data produce maps, create management plans, design field sampling, and do statistical analyses. The part I have the least amount of experience with is building statistically models with spatial data. 
+
+7.  How do you feel about taking this course (e.g., neutral, positive, negative) and why (e.g., aspects you are worried about, or looking forward to)?
+
+Positive! I'm really looking forward to building my R skills, especially with spatial statistics versus traditional statistics that I've had a lot of experience with. Also, I'm excited to learn how to make nice maps in R so I don't have to use ArcGIS.
+
+8.  After reviewing the syllabus, give some thought to your objectives for this course. Which parts of this course are going to be most useful to your research? What skills do you really want to focus on during the semester? Once you've given this some thought, craft 3-5 SMART objectives for you in this course. SMART objectives are Specific (e.g., I want to make maps in R), Measurable (e.g., I want to build 3 maps displaying different elements of a location in R), Achievable (e.g., I want to build 3 maps displaying different elements of a location using ggplot), Relevant (e.g., I want to build 3 maps displaying social and ecological elements of a location using ggplot), and Time-bound (e.g., I want to build 3 maps displaying social and ecological elements of a location using ggplot for the final project). Try to think about your overall objectives for the course so that we can use to assess your final project. Remember, these objectives should be based in where you are now and where you'd like to be when the course ends. If you're not sure how to answer this, schedule a meeting with me to chat about it.
+
+  1) I want to use ggplot to generate publication-quality maps showing ecological qualities of the study location for the final project.
+  
+  2) I want to build a flexdashboard using Quarto and Shiny for reporting my final project products. 
+  
+  4) I want to learn how to build an interactive map in R that effectively communicates spatial data and could be used in a dashboard or website. 
```

Comment:
I'm happy to provide feedback on this if you make one! Are you planning on trying this out in the final project? I believe you can incorporate an interactive map into a flexdashboard with Shiny.